### PR TITLE
Update CODEOWNERS and MAINTAINERS.md

### PR DESCRIPTION
**Pull request by jgarciahospital**
#### What type of PR is this?

* subproject management

#### What this PR does / why we need it:

Including initial list of maintainer/codeowner

### CODE REVIEW

**Review by hdamker — CHANGES_REQUESTED** [review]
Sent invitations to @Masa810 and @yamamoto0104 to get members of the CAMARA organization. Waiting for the acceptance of the invitations.

**Comment by Masa8106** [comment]
@jgarciahospital , @hdamker  I have accepted the invitation. Thank you.

**Comment by yamamoto0104** [comment]
@jgarciahospital , @hdamker , Thank you for updating CODEOWNERS and MAINTAINERS.md.
I also accepted the invitation.

**Comment by ToshiWakayama-KDDI** [comment]
Thanks very much, @jgarciahospital and @hdamker, for processing with this.  

To @Herbert, could you provide Codeowners/Maintaienrs from DT, please, so that we could start working on this API.

Many thanks,
Toshi
KDDI

**Comment by ToshiWakayama-KDDI** [comment]
Hi @jgarciahospital , @hdamker ,

We have just had a KYC SP meeting today, and confirmed that @grgpapadopoulos (Georgios Papadopoulos) is the codeowner from DT.  So, @jgarciahospital , could you add Georgios to the Codeowner file in this PR?  

Then, @hdamker , could you approve and merge the codeowner update?  Or, do you need to merge this PR including both of codeowner and maintainer updates altogether? 

For Maintainers, we are waiting for DT's response, which is expected by the end of today.

Many thanks,
Toshi
KDDI

File: CODEOWNERS
Comment (hdamker) [inline CODEOWNERS:9]:
```suggestion
* @eric-murray @Masa8106 @grgpapadopoulos
```

File: MAINTAINERS.MD
Comment (hdamker) [inline MAINTAINERS.MD:5]:
```suggestion
| KDDI | Kenichi Yamamoto | @yamamoto0104  |
| DT | Georgios Papadopoulos | grgpapadopoulos |
| DT | Noel Wirzius | NoelWirzius |
```

**Comment by jgarciahospital** [comment]
Solved. If @ToshiWakayama-KDDI you feel confortable with current proposal, I think @hdamker that we can merge.

**Comment by jgarciahospital** [comment]
Ready. @ToshiWakayama-KDDI please confirm to that @hdamker can merge.

**Review by hdamker — APPROVED** [review]

**Comment by hdamker** [comment]
> Ready. @ToshiWakayama-KDDI please confirm to that @hdamker can merge.

Thanks @jgarciahospital @ToshiWakayama-KDDI. I've added @grgpapadopoulos and @NoelWirzius to Maintainers.md, so that it is currently on par with the Tenure repository regarding DT. If there are changes they can be done in a new PR.

Here we are done. I'll take https://github.com/camaraproject/NumberRecycling/pull/4#issuecomment-2385130416 as approval from @ToshiWakayama-KDDI (technically I don't need it).

**Pull request merged**
